clinica/iotools/bids_utils.py

```diff
@@ -514,9 +520,12 @@ def write_scans_tsv(bids_dir, bids_ids, scans_dict):
                             for carac in file_name.split("_")
                             if "-" in carac
                         }
-                        f_type = description_dict["acq"].upper()
+                        f_type = description_dict["trc"].upper()
```

Comment:
Why acq -> trc?

Reply:
Discussions with Elina (and Ninon, Mauricio & cie) where the "acq" suffix in the PET files should become "trc".